cmd/move_config.go

```diff
@@ -66,12 +68,18 @@ func Move(dryRun bool) error {
 
 	log.Println("WARNING: Helm v3 configuration may be overwritten during this operation.")
 	log.Println()
-	doCleanup, err := utils.AskConfirmation("Move Config", "move the v2 configuration")
-	if err != nil {
-		return err
+	if skipConfirmation {
+
+		log.Println("Skipping confirmation before performing move.")
+		doCleanup = true
+	} else {
+		doCleanup, err = utils.AskConfirmation("Move Config", "move the v2 configuration")
+		if err != nil {
+			return err
+		}
 	}
 	if !doCleanup {
-		log.Println("Move configuration will not proceed as the user didn't answer (Y|y) in order to continue.")
+		log.Println("Move will not proceed as the user didn't answer (Y|y) in order to continue.")
```

Comment:
No need to change this

Reply:
The print statement is still changed

Reply:
Can you not make this change: `log.Println("Move will not proceed as the user didn't answer (Y|y) in order to continue.")`. In other words, leave the message as it was.